source/includes/_rankings.md

```diff
@@ -538,6 +618,71 @@ curl -X POST \
     https://tools.brightlocal.com/seo-tools/api/v4/rankings/bulk-search
 ```
 
+```csharp
+List<string> searches = new List<string>(
+               new string[] {
+                    "restaurant new york'",
+                    "restaurant manhattan",
+                    "restaurant 10019"
+               });
+api Api = new api("<INSERT_API_KEY>", "<INSERT_API_SECRET>");
+batchApi batchRequest = new batchApi(Api);
+
+// Create a new batch
+int batchId = batchRequest.Create();
+var parameters = new api.Parameters();
+        parameters.Add("batch-id", batchId);
+        parameters.Add("search-engine", "google");
+        parameters.Add("country", "USA");
+        parameters.Add("telephone", "+1 212-554-1515");
+        parameters.Add("google-location", "New York, NY");
+        parameters.Add("search-terms", searches);
+        parameters.Add("urls", "['le-bernardin.com']");
+        parameters.Add("business-names", "['Le Bernardin']");
+        var jobId = Api.Post("/v4/rankings/bulk-search", parameters);
+
+if (jobId.ResponseStatus == ResponseStatus.Completed)
+{
+    dynamic job = JsonConvert.DeserializeObject(jobId.Content);
+    if (job.success != "true")
+    {
+       string message = "Error adding job";
+       var batchException = new ApplicationException(message + job.errors, job.ErrorException);
+       throw batchException;
+    }
+}
+else
+{
+    throw new ApplicationException(jobId.ErrorMessage);
+}
+
+// Commit the batch, resturns true or false
+bool commit = batchRequest.Commit(batchId);
+
+// Poll for results. In a real world example you should do this in a backgroud process, such as HangFire,  or use the Task Parallel Library to create a BackGroundWorker Task.
+// It is bad practice to use Thread.Sleep(). This is only for the example and will actually freeze the UI until the while loop is finished. 
+
+var results = batchRequest.GetResults(batchId);
+dynamic rankingResults = JsonConvert.DeserializeObject(results.Content);
+
+if (rankingResults.success == "true")
```

Comment:
This is "true" as a string. Because I'm checking the success parameter of the results. Should it be boolean? I didn't think that would work. I can certainly try that.